ansible/README.md

```diff
@@ -43,7 +43,8 @@ These playbooks are available to you:
   - **jenkins/linter.yml**: Sets up the code linters (flavour of a worker).
 
   - **create-webhost.yml**: Configures the server(s) that host nodejs.org,
-                            iojs.org and dist.libuv.org among other things.
+                            iojs.org and dist.libuv.org among other things
+                            (only works on Ubuntu boxes).
```

Comment:
@gibfahn not at all; I just think its a "level of knowledge" thing. I mean, should we write operating systems everywhere? If it helps someone that its there I don't mind -- just wanted to state that we have similar variations all over the place.

@santigimeno lets keep it in!